Frontend/ui-driver/src/Services/Endpoint.purs

```diff
@@ -150,3 +150,6 @@ onCall _ = (getBaseUrl "") <> "/onCall"
 
 likeMessage :: String -> String
 likeMessage messageId = (getBaseUrl "") <> "/message/" <> messageId <> "/like"
+
+currentDateAndTime :: String -> String
+currentDateAndTime _ = "https://tools.aimylogic.com/api/now"
```

Comment:
@witcher-shailesh  Can you verify this API Can be used?

Reply:
What is API latency here?